tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -189,6 +189,7 @@ set( test_src
   Vector.unit.cc
   VentilatedSlab.unit.cc
   WaterCoils.unit.cc
+  WaterManager.unit.cc
```

Comment:
Another new unit test file, this is great.  I went to see if there was a clear_state in the water manager, but there isn't -- because the water manager doesn't hold any global state....at least that's how it "appears".  There are a number of static variables inside functions inside WaterManager that could cause the simulation state to go haywire.  I would like to see those extracted into the global state and a new clear_state method that will call it to reset the water manager.  It doesn't necessarily have to be in this PR, but if not, then in a quick follow-up PR.  And if you aren't able to accommodate it right now, I'll open a PR and do it.